contracts/marketplace/Marketplace.sol

```diff
@@ -103,85 +113,122 @@ contract Marketplace is Ownable, Pausable, Destructible {
   /**
     * @dev Cancel an already published order
     * @param assetId - ID of the published NFT
-    * @param priceInWei - Price in Wei for the supported coin.
-    * @param expiresAt - Duration of the auction (in hours)
+    * @param nftAddress - Non fungible registry address
+    * @param priceInWei - Price in Wei for the supported coin
+    * @param expiresAt - Duration of the order (in hours)
     */
-  function createOrder(uint256 assetId, uint256 priceInWei, uint256 expiresAt) public whenNotPaused {
-    address assetOwner = nonFungibleRegistry.ownerOf(assetId);
-    require(msg.sender == assetOwner);
-    require(nonFungibleRegistry.isAuthorized(address(this), assetId));
-    require(priceInWei > 0);
-    require(expiresAt > block.timestamp.add(1 minutes));
-
-    bytes32 auctionId = keccak256(
-      block.timestamp, 
-      assetOwner,
-      assetId, 
-      priceInWei
+  function createOrder(
```

Comment:
I wonder if we should use the same order for parameters in this function() as in executeOrder() and cancelOrder() -> nftAddress then assetId

What do you think?